File: app/src/main/java/com.example.petridelivery.wrappers/base/abs/JsonMapper.java
package com.example.petridelivery.wrappers.base.abs;

public interface JsonMapper {
	<T> T fromJson(String json, Class<T> cls);
	String toJson(Object obj);
	<T> T fromJson(File f, Class<T> cls);
	void toJson(File f, Object obj);
}

File: app/src/main/java/com.example.petridelivery.wrappers/base/abs/JsonMapper.kt
package com.example.petridelivery.wrappers.base.abs

interface JsonMapper {
    fun <T> fromJson(json: String?, cls: Class<T>): T?
    fun toJson(obj: Any?): String
    fun <T> fromJson(f: File, cls: Class<T>): T?
    fun toJson(f: File, obj: Any?)
}